openapi.yml

```diff
@@ -14,9 +14,15 @@ paths:
         '200':
           description: The status of the service
   /v1/dataloads:
-    get:
+    post:
       summary: Retrieve the list of scheduled data loading jobs
       description: Provide a list of jobs to run since the last request
```

Comment:
I'm having trouble grokking this endpoint. Is the last request persisted in the app? How does it related to the provided date?

Reply:
My vision for this, and I'm more than open to suggestion, is that the date provided will be used to select all `dataload_jobs` that have a status of `scheduled` with a date earlier than the provided date. This should provide all scheduled jobs that should be queued as well as any immediate jobs since the last request. Each job is then updated with a `queued` status so that it is not selected again with the next request form airflow.

Reply:
Why is the date parameter needed? Why not just provide all `scheduled` jobs?

Just trying to understand the proposed approach and want to make sure the API docs make the approach clear.

Reply:
I may be overthinking this, I believe we want this endpoint to be a POST because it will effect records, however we can use the `datetime` of the request, it doesn't need to be in the payload - that does seem to complicate it a bit.

Reply:
@justinlittman updated to make this a POST, but removed the payload.